docs/proposals/15-manage-bastions-and-ssh-key-pair-rotation.md

```diff
@@ -222,3 +222,8 @@ Currently, the `ssh` key pair for the shoot nodes are created once during shoot
         - The "execution" script includes also the original user data script, which it writes to `PATH_CLOUDCONFIG`, compares it against the previous cloud config and runs the script in case it has changed
         - Running the [original user data](https://github.com/gardener/gardener/tree/master/pkg/operation/botanist/component/extensions/operatingsystemconfig/original) script will also run the `gardeneruser` component, where the `authorized_keys` file will be updated
         - After the most recent cloud-config user data was applied, the "execution" script annotates the node with `checksum/cloud-config-data: <cloud-config-checksum>` to indicate the success
+
+### Limitations
+
+For some cloud providers, like GCP, SSH keypairs are managed at the provider side and not on the seed cluster as Kubernetes
```

Comment:
This is not correct. The SSH keys are always stored in the seed clusters as secrets and also synced to the garden cluster. 

Simply, on GCP the VMs do not have any static users and there is an agent on the nodes that syncs the users with their ssh keys from the GCP IAM service.

Reply:
Perfect, let me just quote you directly. :)

Reply:
I think this is missing the part where it mentions that the SSH keys of the default OS users are currently not rotated.

> Each OS has its own default user, e.g. core, admin, ec2-user, etc. These users get their ssh keys during VM creation.